app/controllers/NextUpdatesController.scala

```diff
@@ -77,8 +77,7 @@ class NextUpdatesController @Inject()(NoNextUpdatesView: NoNextUpdates,
             auditNextUpdates(user, isAgent, origin)
 
             val optOutSetup = for {
-              checks <- optOutService.getNextUpdatesQuarterlyReportingContentChecks
-              optOutOneYearViewModel <- optOutService.nextUpdatesPageOptOutViewModel()
+              (optOutOneYearViewModel, checks) <- optOutService.nextUpdatesPageOptOutWithChecks()
```

Comment:
not covered by unit test
![Screenshot 2024-07-04 at 11 28 32](https://github.com/hmrc/income-tax-view-change-frontend/assets/165788042/3f54c1a7-0fd9-439a-bdb0-6d702b75e01c)